Bellerophon/Bellerophon.xcodeproj/xcshareddata/xcschemes/Bellerophon.xcscheme

```diff
@@ -26,6 +26,7 @@
       buildConfiguration = "Debug"
       selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
+      language = ""
```

Comment:
Did this need to change?

Reply:
I didn't make the change myself. Xcode might have done it since I'm using the latest Xcode